koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -351,101 +239,10 @@ def osbs(self):
             assert self._osbs
         return self._osbs
 
-    def _rpm_package_info(self, parts):
-        if len(parts) < 8:
-            self.logger.error("Too few number of fields in the list"
-                              " of rpms: %r", parts)
-            return
-        rpm = {
-            'name': parts[0],
-            'version': parts[1],
-            'release': parts[2],
-            'arch': parts[3],
-            'size': int(parts[5]),
-            'sigmd5': parts[6],
-            'buildtime': int(parts[7])
-        }
-
-        if parts[4] == '(none)':
-            rpm['epoch'] = None
-        else:
-            rpm['epoch'] = int(parts[4])
-        return rpm
-
-    def getUploadPath(self):
-        """Get the path that should be used when uploading files to
-        the hub."""
-        return koji.pathinfo.taskrelpath(self.id)
 
     def resultdir(self):
         return os.path.join(self.workdir, 'osbslogs')
 
-    def _incremental_upload_logs(self, child_pid):
```

Comment:
I had previously removed the code that was using it and nobody seemed concerned so it was basically dead code and I pulled it out, but you're probably right and we'll just need to figure out how best to handle that now.